react/features/toolbox/middleware.js

```diff
@@ -7,6 +7,48 @@ import {
     SET_TOOLBOX_TIMEOUT
 } from './actionTypes';
 
+import {
+    SET_VIDEO_AVAILABLE,
+    SET_VIDEO_MUTED
+} from '../../features/base/media/actionTypes';
+
+import {
+    setToolbarButton
+} from './actions.native.js';
```

Comment:
That would work because index.js re-exports actions.js. Another way could be `import { setToolbarButton } from './actions'`.